website/docs/language/settings/terraform-cloud.mdx

```diff
@@ -28,6 +28,9 @@ terraform {
 }
 ```
 
-You cannot use the CLI integration and a [state backend](/language/settings/backends) in the same configuration; they are mutually exclusive. A configuration can only provide one `cloud` block and the `cloud` block cannot refer to named values like input variables, locals, or data source attributes.
+You cannot use the CLI integration and a [state backend](/language/settings/backends) in the same configuration; they are mutually exclusive.
+A configuration can only provide one `cloud` block and the `cloud` block cannot refer to named values like input variables, locals, or data source attributes.
+However, if a state `backend` block is defined in an [override file](/language/files/override),
+that `backend` block will override an existing `cloud` block in the configuration upon merging.
```

Comment:
I don't think this page actually needs this information -- as far as I know, we don't interject with details about `_override.tf` file block merging for any other block types, and I think it makes sense to keep that confined to just the override files page.

Reply:
Yeah that makes a lot sense, have made this change.